packages/tools/generator-widget/generators/app/index.js

```diff
@@ -7,6 +7,9 @@ const text = require("./lib/text.js");
 
 const widgetSrcFolder = "src/components/";
 
+// Extends the prototype as install actions was removed from yeoman-generator
```

Comment:
it seems strange that they deprecated action and didn't provide a way to do it in a newer version. :/

[Yeoman issues](https://github.com/yeoman/generator/issues/1294) 
People already complaining about it but seem's it's a documentation problem, it's implemented somewhere in the code.
I don't have too much context about it, but maybe for you, it's worth to check

Reply:
Well, they do, and I added their solution extending the prototype.